mountpoint-s3-client/Cargo.toml

```diff
@@ -7,7 +7,6 @@ publish = false
 
 [dependencies]
 mountpoint-s3-crt = { path = "../mountpoint-s3-crt" }
-mountpoint-s3-crt-sys = { path = "../mountpoint-s3-crt-sys" }
```

Comment:
Why is mountpoint-s3-crt-sys removed as a dependency for s3-client. Is it because it only depends directly on mountpoint-s3-crt? Or there is some other reason?

Reply:
Yeah, there is no reason for the `mountpoint-s3-client` to depend on the `mountpoint-s3-crt-sys` crate, which is providing the C/Rust bindings to the CRT.

Instead, the `mountpoint-s3-client` should only care about the Rust-ified interface provided by `mountpoint-s3-crt`.